pages/projects.js

```diff
@@ -5,24 +5,18 @@ import { getProjects } from '../api/projects';
 import { Label } from '../components/label';
 
 const ProjectsPage = ({ page, menu, footer }) => {
-  console.log(page);
   return (
     <Layout title="Göker Blog - My projects" footer={footer} menu={menu.menuItems}>
       <Container width="960px">
-        <div>
-          {page.list.map((data) => (
-            <ProjectContent
-              key={data.id}
-              banner={data.content.thumbnail}
-              title={data.content.title}
-              usedTechnology={data.content.technologies.map((tech) => (
-                <Label usedTechnologies={tech} />
-              ))}
-              summary={data.content.description}
-              projectlink={data.content.projectLink}
-            />
-          ))}
-        </div>
+        {page.list.map((data) => (
+          <ProjectContent
+            key={data.id}
+            projects={data.content}
+            usedTechnology={data.content.technologies.map((tech) => (
```

Comment:
we don't need use render prop here so we can remove `usedTechnology`since data.content has already the technologies